modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/utils/SystemCmd.java

```
@@ -68,24 +68,23 @@ public boolean execute(String cmd, StringBuilder stdErrOut, long waitSecond) {
                 process.waitFor();
             } else {
                 if (!process.waitFor(waitSecond, TimeUnit.SECONDS)) {
-                    log.error("SystemCmd wait process failed. {}", String.join(" ", cmd));
-                    throw new IllegalStateException("SystemCmd wait process failed. \nCmd:\n" + cmd
+                    log.error("SystemCmd wait process failed. \nCmd:\n" + cmd
                             + "\nOutput:\n" + stdErrOut.toString());
+                    return false;
                 }
             }
             if (process.exitValue() != 0) {
-                log.error("SystemCmd process finished with abnormal value.");
-                throw new IllegalStateException(
-                        "SystemCmd process finished with abnormal value. \nCmd:\n" + cmd
-                                + "\nOutput:" + stdErrOut.toString());
+                log.error("SystemCmd process finished with abnormal value. \nCmd:\n" + cmd
+                        + "\nOutput:" + stdErrOut.toString());
```

we should throw the exception here.